extensions/warp-ipfs/src/js_exports.rs

```diff
@@ -0,0 +1,180 @@
+use std::str::FromStr;
+
+use crate::WarpIpfs;
+use js_sys::Uint8Array;
+use warp::crypto::DID;
+use warp::multipass::{
+    identity::{self, Identity, IdentityProfile},
+    MultiPass,
+};
+use wasm_bindgen::prelude::*;
+
+#[wasm_bindgen]
+pub struct MultiPassBox {
+    inner: Box<dyn MultiPass>,
+}
+#[wasm_bindgen]
+impl MultiPassBox {
+    pub fn new(instance: WarpIpfs) -> Self {
+        Self {
+            inner: Box::new(instance.clone()) as Box<_>,
```

Comment:
Clone looks redundant

```suggestion
            inner: Box::new(instance) as Box<_>,
```

Reply:
nice catch. done